src/convex/notionSync/identity.ts

```
@@ -0,0 +1,42 @@
+import {v} from 'convex/values';
+import type {Doc} from '../_generated/dataModel';
+import {internalQuery} from '../_generated/server';
+import {getNotionSyncFallbackUserExternalId} from '../notion/config';
+import type {NotionPage} from '../notion/types';
+import {getUserByExternalId, getUserByNotionUserId} from '../users';
+
+type ObjectOwnerCandidate = Pick<Doc<'users'>, 'isDeleted' | 'notionSyncEnabled'>;
+
+export function getNotionPageOwnerId(
+    page: Pick<NotionPage, 'created_by' | 'last_edited_by'>,
+) {
```

ESLint

[error] Replace `⏎····page:·Pick<NotionPage,·'created_by'·|·'last_edited_by'>,⏎` with `page:·Pick<NotionPage,·'created_by'·|·'last_edited_by'>`

(prettier/prettier)